### ISSUE TITLE
Change puzzle favicon to a Firefox favicon

### ISSUE THREAD

**Issue by mlissner** [mozilla/addons#632]
### Describe the problem and steps to reproduce it:

Currently, this is what the Opera, ~~Firefox~~ (this screenshot is outdated), and Chrome favicons look like:

![screenshot from 2018-01-31 09-33-19](https://user-images.githubusercontent.com/236970/35637695-cb13d9d4-0669-11e8-835f-1ef2f93b21af.png)

The Opera and Chrome favicons link back to their branding nicely, but the Mozilla icon is kind of...meaningless. Add-ons are....like puzzle pieces, I guess? 

The tricky thing is that the store has addons for Firefox and Thunderbird, but as somebody that regularly hops between all three stores, I've totally noticed that the favicon for AMO doesn't click in the way the other icons do. 

### What happened?

1. I looked for the AMO store and had to stare at the three icons for a second to figure out which was which. 
1. I looked for the Opera store, and it was obvious which tab to click.

### What did you expect to happen?

I'd expect the Firefox logo to be prominent in AMO. What percent of add-ons in the store are for Firefox?

**Comment by kumar303** [comment]
@mlissner your screenshot is for the favicon of a legacy page that hasn't been ported to the new site yet (see https://github.com/mozilla/addons-frontend/issues/2749). As Matt mentioned, we do have a meaningful favicon.

**Comment by mlissner** [comment]
Wait, that explains why it's the old icon, but all of my arguments still apply to the new favicon too, which is still a puzzle piece, just turned 45 degrees?

**Comment by kumar303** [comment]
I dunno, I like the new puzzle icon. It gives the site personality and makes it stand out from other Mozilla tabs which have a Firefox favicon ([example](https://accounts.firefox.com/settings)). Thanks for taking the time to file an issue but the favicon is intentional.

**Comment by mlissner** [comment]
Don't get me wrong, it's a decent icon, I'm just sure that it has no meaning for me when I'm looking for AMO. E.g., is that icon used anywhere else in the site? I don't usually have other Firefox tabs open since I'm not a FF developer.

The only meaning I consistently get from the new icon is that it looks like [Maus](https://en.wikipedia.org/wiki/Maus):

![maus_volume_2_page_50_panels_3-4](https://user-images.githubusercontent.com/236970/35645702-9089eb92-0681-11e8-8e40-86c2adffdd79.png)

**Comment by pwalm** [comment]
ooh, Maus is so great. Worth a revisit.

I was checking out some Mozilla and Firefox-related sites today and it seems we don't have a unified approach for favicons. Using the Firefox glyph makes the most sense, I think. We can still use the puzzle piece icon for extensions that don't have an icon.

**Comment by kumar303** [comment]
Thanks for weighing in, @pwalm. I'll re-open to change the puzzle favicon to a Firefox favicon.

See https://design.firefox.com/photon/visuals/product-identity-assets.html

**Comment by eviljeff** [comment]
@kumar303 can you find the issue where the puzzle piece was first introduced?  (I'm sure it was only recently, and much discussed at the time).  Seems we're changing direction (again) on a whim.

**Comment by mlissner** [comment]
FWIW, there doesn't seem to be an issue on Github relating to favicons that I can find.

**Comment by bobsilverberg** [comment]
Moved this to https://github.com/mozilla/addons-frontend/issues/4407

**Comment by bobsilverberg** [comment]
I did some work on this today, trying to get webpack to write the new favIcon file as an asset and then use the address from webpack in ServerHtml.js. There was a problem that the `import` I had to add to ServerHtml.js was throwing an error, because webpack didn't seem to be using a loader to process it. Some testing revealed that the same import did work in a different component (e.g., the <Addon> component), so it does point to an issue with ServerHtml.js.

I am attaching a patch with my changes thus far so this issue can be (somewhat) easily picked up again when I or someone else has the time.

[output.txt](https://github.com/mozilla/addons/files/1732278/output.txt)
